1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -13,6 +13,12 @@ let salaries = {
 };
 
 // WRITE BELOW THIS LINE
+for( let x in salaries){  
```

Comment:
Good. The outn put is not quite correct. It needs to include the key and the value